src/utils/RouteUtils.js

```diff
@@ -70,6 +70,9 @@ async function getRoutes(
         destinationName,
     );
 
+    // Ensure that arrivalTime and departureTime have at least one minute difference
+    ParseRouteUtils.adjustRouteTimesIfNeeded(walkingRoute);
```

Comment:
isn't this from @kevinchan159  changes? can we merge `master` into this branch so this doesn't show up in your changes?

```
git checkout master
git pull
git checkout [YOUR BRANCH]
git merge master # merges master into the current branch
```